### ISSUE TITLE
Autoupdater doesn't close/open the app automatically

### ISSUE THREAD

**Issue by TinyRaindrop** [Soundofdarkness/RuneBook#48]
It shows the "App will restart for the update" alert, but I have to manually close and reopen it.

**Comment by Soundofdarkness** [comment]
Yep noticed this too, no idea why, I will see if I can find the reason

**Comment by Miladiir** [comment]
https://github.com/electron-userland/electron-builder/issues/5101

Maybe try a different version of the dependency? Not sure why, but they have two newer versions tagged but not as `latest`.
